src/core/crypto.c

```diff
@@ -855,11 +855,16 @@ int32_t Crypto_Check_Anti_Replay(SecurityAssociation_t* sa_ptr, uint8_t* arsn, u
         // Valid ARSN received, increment stored value
         else
         {
-            ARSN_VALID = CRYPTO_TRUE;
+            *ARSN_VALID = CRYPTO_TRUE;
             // memcpy(sa_ptr->arsn, arsn, sa_ptr->arsn_len);
         }
     }
-    // If IV is greater than zero and using GCM, check for replay
+    return status;
+}
+
+int32_t Crypto_Check_Anti_Replay_GCM(SecurityAssociation_t* sa_ptr, uint8_t* iv, int8_t* IV_VALID)
```

Comment:
Should IV_VALID be lowercase since it's not a #define?

Reply:
It absolutely should be.  I didn't write this portion of the code.  I was just reducing complexity.  Do we want to fix these minor issues in this large merge as well?

Reply:
Fixed -- in upcoming push.